Don't find spectre, poppler, or librsvg even if they're installed.
https://trac.macports.org/ticket/33856
https://lists.macports.org/pipermail/macports-dev/2012-March/018411.html
--- configure.orig	2022-03-18 13:40:22.000000000 -0500
+++ configure	2023-02-03 20:39:58.000000000 -0600
@@ -25179,7 +25179,7 @@
 
 
 # Keep in sync with meson.build!
-SPECTRE_VERSION_REQUIRED=0.2.0
+SPECTRE_VERSION_REQUIRED=99999
 test_ps=no
 any2ppm_ps=no
 if test "x$use_ps" = "xyes"; then
@@ -25573,7 +25573,7 @@
 
 # poppler-0.17.4 fixes text-pattern and text-transform
 # Keep in sync with meson.build!
-POPPLER_VERSION_REQUIRED=0.17.4
+POPPLER_VERSION_REQUIRED=99999
 test_pdf=no
 any2ppm_pdf=no
 if test "x$use_pdf" = "xyes"; then
@@ -25939,7 +25939,7 @@
 
 
 
-LIBRSVG_VERSION_REQUIRED=2.35.0
+LIBRSVG_VERSION_REQUIRED=99999
 test_svg=no
 any2ppm_svg=no
 if test "x$use_svg" = "xyes"; then
